services/apps/premium/members_enrichment_worker/src/workflows/processMemberSources.ts

```
@@ -0,0 +1,287 @@
+
+export async function processMemberSources(args: IProcessMemberSourcesArgs): Promise<boolean> {
+  // without contributions since they take a lot of space
+  const toBeSquashed = {}
+
+  let hasContributions = false
+
+  // find if there's already saved enrichment data in source
+  const caches = await findMemberEnrichmentCache(args.sources, args.memberId)
+  for (const source of args.sources) {
+    const cache = caches.find((c) => c.source === source)
+    if (cache && cache.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)